chart/kube-flannel/tests/daemonset_test.yaml

```diff
@@ -0,0 +1,47 @@
+suite: test daemonset
```

Comment:
How does this test work? Do we have to package it inside the chart or we can skip it with `.helmignore`?

Reply:
Should I do something about it?

Reply:
No it's ok.